### DISCUSSION TITLE
Widget in not using theme

### DISCUSSION THREAD

**Discussion by webmedia1012** [OrchardCMS/OrchardCore#11087] [General]
Hi all,

I'm having trouble understanding on how you can hook widget created in admin on to a razor page, I do understand that if you use a theme (blog for instance) it would be automatically created, but I didn't use theme, this CMS was blank when I created it, but what if I am using the injected IOrchardHelper ? how do I invoke the widget?, what's the method you used ? is it by content type as for example GetContentItemByHandleAsync.

Thank you as always,
G

**Comment by ns8482e** [comment]
widget created for layers use layer rules to render widget and layers only work with display management
